network-api/networkapi/wagtailpages/pagemodels/mixin/foundation_metadata.py

```diff
@@ -72,3 +72,11 @@ def get_meta_image(self):
 
     class Meta:
         abstract = True
+
+
+# Overriding default name and help text for the promote panels 'search_image' field.
```

Comment:
Since every page inherits from the `FoundationMetaDataPageMixin`, I have added the overrides to this file so the updates can be sitewide.